### ISSUE TITLE
Nested model not included in Pydantic model

### ISSUE THREAD

**Issue by pawamoy** [collerek/ormar#278]
**Describe the bug**
I don't know if it comes from FastAPI, Pydantic or Ormar. When retrieving a Pydantic model from an Ormar model, the response example shown in the Swagger interface does not include a nested model. Same with the actual responses: they do not contain that nested model.

My models:

```python
# meta classes removed for conciseness

class Library(ormar.Model):
    id: int = ormar.Integer(primary_key=True)
    name: str = ormar.String(max_length=100)


class Package(ormar.Model):
    id: int = ormar.Integer(primary_key=True)
    library: Library = ormar.ForeignKey(Library, related_name="packages")  # <-- of interest
    version: str = ormar.String(max_length=100)


class Ticket(ormar.Model):
    id: int = ormar.Integer(primary_key=True)
    number: int = ormar.Integer()
    status: str = ormar.String(max_length=100)


class TicketPackage(ormar.Model):
    id: int = ormar.Integer(primary_key=True)
    status: str = ormar.String(max_length=100)
    ticket: Ticket = ormar.ForeignKey(Ticket, related_name="packages")
    package: Package = ormar.ForeignKey(Package, related_name="tickets")  # <-- of interest
```

My route:

```python
router = APIRouter(prefix="/tickets")
TicketPackageOut = TicketPackage.get_pydantic(exclude={"ticket"})


@router.get(
    "/{ticket_id}/packages",
    response_model=List[TicketPackageOut],
)
async def get_ticket_packages(ticket_id: int) -> List[TicketPackage]:
    return await TicketPackage.objects.select_related("package__library").filter(ticket__id=ticket_id).all()
    # Note how I "select the related package library".
    # If I print the return value of that query,
    # I can see that the library objects are here
    # with all their values (name) populated, as expected.
```

**Expected behavior**

The example and actual responses in Swagger should look like:

```json
[
  {
    "id": 0,
    "status": "string",
    "package": {
      "version": "string",
      "id": 0,
      "library": {
        "id": 0,
        "name": "string"
      }
    }
  }
]
```

**Actual behavior**

The responses look like:

```json
[
  {
    "id": 0,
    "status": "string",
    "package": {
      "version": "string",
      "id": 0
    }
  }
]
```

The package library is missing!

**Screenshots**

Example response | Actual response
--- | ---
![image](https://user-images.githubusercontent.com/3999221/126457728-6087270b-22f6-43aa-b049-862c03f7a76c.png) | ![image](https://user-images.githubusercontent.com/3999221/126457883-9725c888-59ce-4993-8bb6-ea675a80691b.png)

*You'll notice extra fields that I removed from the code above for conciseness.*

**Versions:**
 - Database backend used: sqlite
 - Python version: 3.8.11
 - `ormar` version: 0.10.14
 - `pydantic` version: 1.8.2
 - if applicable `fastapi` version: 0.65.2

**Comment by collerek** [comment]
Can you check if it's there when you set the response model simply to `TicketPackage`?
There might be a bug in the pydantic model generation.

**Comment by pawamoy** [comment]
It is, yes!

I'm currently debugging: it seems that when generating the model, and reaching/recursing on the `package` field, the new relation map is computed using `relation_map=cls._skip_ellipsis(relation_map, ...)`, and since `library` *is* an ellipsis, the new relation map becomes empty when handling the `library` field.

Then, `_determine_pydantic_field_type` fails to return a proper field (it returns `None` for `library`) because of `if field.is_relation and name in relation_map:`.

In the end, `library` is not included in the final model because of `if field is not None:`.

**Comment by pawamoy** [comment]
In fact, `cls._skip_ellipsis` gets passed the `field`, which is then used as a key to get the result from the items, which is `relation_map`. Then `cls.get_child` actually runs `return items.get(key, {})`, and since the field cannot be found in the relation map, it returns an empty dict. So the fact that `library` is an ellipsis is not a problem. It seems the issue is that you pass the field instead of its name to `cls._skip_ellipsis`? But I just started looking at the code so I'm probably wrong 😅

**Comment by pawamoy** [comment]
Well this diff seems to fix the issue!

```diff
                 relation_map=cls._skip_ellipsis(
-                    relation_map, field, default_return=dict()
+                    relation_map, name, default_return=dict()
                 ),
```

**Comment by collerek** [comment]
Great catch, you want to issue a pr? 
I just pushed a branch `bug_fixes` with tests based on your code ;)
So you can fork from it and apply changes.

**Comment by pawamoy** [comment]
Sure, you make it so easy, I cannot refuse 😁

**Comment by collerek** [comment]
Fixed by #280
